shared/signup/common.tsx

```diff
@@ -7,9 +7,10 @@ import openURL from '../util/open-url'
 import * as Styles from '../styles'
 
 type InfoIconProps = {
+  loggedIn: boolean
   invisible?: boolean
   onDocumentation: () => void
-  onFeedback: () => void
+  onFeedback: (loggedIn: boolean) => void
```

Comment:
:sheep: you can pass this in `mergeProps` in the connector instead of having to plumb `loggedIn` through the component.